### DISCUSSION TITLE
Rerun command when loading editor

### DISCUSSION THREAD

**Discussion by mopg** [ueberdosis/tiptap#2616] [General Questions]
I have a few custom extensions that pull in external data. Now I don't want to save that data in the tiptap document, but rather want to save in the json how to get the data (externally), through attributes. This all works well when running the extension the first time. However, I can't figure out how to pull in the data on loading of the document. Is there a "refresh" command or something like that, that can be run when the editor is reloaded?

Btw love the library!

**Comment by philippkuehn** [comment]
I'm not sure if I understand the issue correctly but you can try to force a re-render with dispatching a noop transaction:

```js
editor.view.dispatch(editor.state.tr)
```

**Reply by mopg** [reply]
Ok great, let me try that. How would I call this on start-up?
